R/z_animintHelpers.R

```diff
@@ -147,6 +147,27 @@ getWidthAndHeight <- function(theme){
   options_list
 }
 
+getRowAndColumn <- function(theme){
+  options_list <- list()
+  for(rc in c("row", "col")){
+    arc <- paste0("animint.", rc)
+    options_list[[rc]] <- if(arc %in% names(theme)){
+      theme[[arc]]
+    }
+  }
+  options_list
+}
+
+getRowspanAndColumnspan <- function(theme){
+  options_list <- list()
+  for(rscs in c("rowspan", "colspan")){
```

Comment:
the only difference between these two functions is the c("row","col") vs c("rowspan","colspan") so please refactor to emphasize that similarity. I would suggest just deleting both of the functions and instead using something like:
```r
for( theme_attribute in c("row","col","rowspan","colspan" ){
  animint.attribute <- paste0("animint.", theme_attribute)
  plot.info[[theme_attribute]] <- plot$theme[[animint.attribute]]
}
```

Reply:
okay. I would do that.

Reply:
merged the two functions into one